andela/zinnia-ah-backend
# [Feature 164797168] Added functionality that enables users edit their comments and track comment history

#### What does this PR do?
Set track and edit comment functionality

#### Description of Task to be completed?
- [x] setup endpoint for editing comments
- [x] setup  endpoint for tracking comment history

#### How should this be manually tested?
* Clone the repo,
* Run ```npm install``` to install all dependencies,
* Create .env file using the .env.example as a guide and enter you environment variables
* Run `sequelize db:create`  
* Run `sequelize db:migrate` 
* Run `sequelize db:seed:all`   
* Run npm run start:dev,
* send a POST request to `localhost:3000/api/v1/articles/:articleId/comments/:commentId/edit` on Postman
* You should get the screenshot below

#### Any background context you want to provide?
 You need a token to have access to the route

#### What are the relevant pivotal tracker stories?(if applicable)
[#164797168](https://www.pivotaltracker.com/story/show/164797168)
#### Screenshots
![image](https://user-images.githubusercontent.com/38259391/56347876-293baa00-61bd-11e9-9c51-521bf956554c.png)
